dockerfiles/hpcc/values.schema.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "global": {
+      "$ref": "#/definitions/global"
+    },
+    "dali": {
+      "description": "dali process",
+      "type": "array",
+      "items": { "$ref": "#/definitions/dali" }
+    },
+    "eclccserver": {
+      "description": "eclccserver process",
+      "type": "array",
+      "items": { "$ref": "#/definitions/eclccserver" }
+    },
+    "esp": {
+      "description": "esp process",
+      "type": "array",
+      "items": { "$ref": "#/definitions/esp" }
+    },
+    "eclagent": {
+      "description": "eclagent process",
+      "type": "array",
+      "items": {
+        "allOf": [
+          { "$ref": "#/definitions/eclagent" },
+          { 
+            "name": {
```

Comment:
Done this way because name is not required (and indeed should ideally be disallowed) on the eclagent section of a thor